### DISCUSSION TITLE
Unable to create new clients/invoices/products

### DISCUSSION THREAD

**Discussion by ironiemix** [baimard/gestion#134] [General]
Hi there, 

thanks a lot for your work on this plugin. I try to use it, but I ran into a dead end and am suspecting that I am missing something crucial.

I entered the information for my "enterprise", that worked o.k., but when I go to the customers entry,  the table says 'No data available in table" and I am unable to create a customer, when I click on "Add customer" nothing happens. 

The same happens in the other sections, the tables all say 'No data available in table" and I am unable to add any entrys.

In addition, when I open the invoice or product page I get an error saying

```
405 Method Not Allowed

Method Not Allowed
The requested method PROPFIND is not allowed for this URL
```

What am I missing here? Any help would be greatly appreciated, TIA, 

Frank

- Nexcloud Hub 23.0.0
- Gestion 2.0.1

**Comment by baimard** [comment]
Dear Ironiemix

Thank you for repporting, I really don't understand what happend.

Could you tell me if the tables with prefix "gestion" have been created ?

If it's possible could you take me an access to your instance ? (user profile)

continue conversation here : https://github.com/baimard/gestion/issues/135
